lib/tbot/config/config_destination.go

```diff
@@ -78,6 +78,37 @@ func (kc *KubernetesCluster) CheckAndSetDefaults() error {
 	return nil
 }
 
+// AppConfig is a cert request for app access.
+type AppConfig struct {
```

Comment:
This will be referred to as `config.AppConfig`. Would `config.App` make more sense?

Reply:
Yeah, that's better. I'd wanted it to match `config.DatabaseConfig` which... well, that should probably be renamed instead. I'll rename both to make everything consistent.